pkg/arduino_adafruit_sensor/patches/0004-Adafruit_Sensor.h-drop-unused-private-field.patch

```diff
@@ -0,0 +1,27 @@
+From 0a165527f2ef58b593ecd95a0ebfc014312d32f7 Mon Sep 17 00:00:00 2001
+From: Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+Date: Fri, 19 May 2023 23:11:05 +0200
+Subject: [PATCH] Adafruit_Sensor.h: drop unused private field
+
+This fixes compilation with clang.
+---
+ Adafruit_Sensor.h | 3 ---
+ 1 file changed, 3 deletions(-)
+
+diff --git a/Adafruit_Sensor.h b/Adafruit_Sensor.h
+index 92e35c2..9681391 100755
+--- a/Adafruit_Sensor.h
++++ b/Adafruit_Sensor.h
+@@ -183,9 +183,6 @@ public:
+   virtual void getSensor(sensor_t *) = 0;
+ 
+   void printSensorDetails(void);
+-
+-private:
+-  bool _autoRange;
```

Comment:
Would upgrading the pkg to [1.1.9](https://github.com/adafruit/Adafruit_Sensor/commit/967e6fab4a37e6298fbf56a8b97de5a369fc0735) fix this?

Reply:
No, [it's still there](https://github.com/adafruit/Adafruit_Sensor/blob/9096b2965335b7373bd2380b765b740da71e2dda/Adafruit_Sensor.h#L220C1-L221C19) and [still not used](https://github.com/search?q=repo%3Aadafruit%2FAdafruit_Sensor%20_autoRange&type=code)